setup.cfg

```diff
@@ -37,7 +37,7 @@ include = usort/*
 omit = usort/tests/*
 
 [coverage:report]
-fail_under = 70
+fail_under = 92
 precision = 1
```

Comment:
make this `precision = 0` and see if that fixes the overly-pedantic failures on 3.8/3.9?